### DISCUSSION TITLE
Bulk add environment variables

### DISCUSSION THREAD

**Discussion by stuartmaxwell** [coollabsio/coolify#1740] [Q&A] [answered]
There should be a feature to bulk add environment variables.

I'm currently testing out Coolify but I have some services that have 20 or more environment variables and adding these one-by-one in each environment is a tedious task. I can't see any way to bulk add these and have searched Github issues, these discussion forums, as well as the docs - hope I haven't missed something.

Thanks for your work on Coolify.

**Comment by andrasbacsai** [comment, marked as answer]
Did you see that there is a developer view of the environment variables? Is this what are you looking for?

![image](https://github.com/coollabsio/coolify/assets/5845193/cf3a0065-aa29-44e1-bfb7-2a4890a44e84)

**Comment by stuartmaxwell** [comment]
Ah - interesting. I guess that is what I was looking for. I had been creating environment variables at the environment level which only lets you add one at a time. But doing it at the resource level makes more sense in my scenario since I only have one resource (a single docker compose file) per environment.
